docs/source/intro/get_started.rst

```diff
@@ -1,69 +1,134 @@
-.. _intro/get_started:
+.. _intro:get_started:
 
 ****************
 Getting Started
 ****************
 
-`#4026 <https://github.com/aiidateam/aiida-core/issues/4026>`_
+A working AiiDA installation consists of three core components, plus any external codes you wish to run:
 
-.. admonition:: Want to jump straight to the tutorials?
+* ``aiida-core``: The main python package and associated CLI ``verdi``.
+* |PostgreSQL|: A service which manages the database where we store generated data.
+* |RabbitMQ|: A service which manages communication with the processes that we run.
 
-    .. container:: link-box
+AiiDA provides a number of routes to setting up this environment, depending on your use case.
+These are outlined below, followed by a recommended "quick-install" route on your local computer.
 
-        Launch AiiDA with MyBinder
+.. panels::
+   :body: bg-light
+   :footer: bg-light border-0
 
+   :fa:`desktop,mr-1` **Direct (Bare Metal)**
 
-Installation
-============
+   *Install software directly into your local root directory.*
```

Comment:
Does "root directory" here imply that the "Direct" installation method is different from installing *via* PyPI or conda? What is a pip/conda installation with only local directories then?

Especially using conda, it is possible (and should be encouraged) to install the system services in user space. I think it should be very clear that the first panel is essentially equivalent to following the rest of the instructions on this page.

Reply:
The key difference IMO between the first and second panel is that, with a direct install you can only install a **singular** instance of the software (well technically a single instance per user, but let's not complicate matters further!).

Going the conda route, means that, yes you install conda itself "directly", but then **everything** else is installed in a virtual environment (including postgres and rabbitmq). This IMO is not equivalent to the first panel, it is primarily the second panel.

With only pip; you have to install postgres and rabbitmq directly. Then you could either use pip directly to install python packages, which would be a "full" direct installation (discouraged), or you can use venv/pipenv to install the python packages in an isolated folder, which is kind of 50/50 between panel one and two.

I have re-worded slightly in 9c0599e to maybe make this clearer

Reply:
Since using conda or pip in a venv is what we recommend, should we make that the first panel then? A direct "bare metal" installation into should be either the second or potentially even the last option IMO.

Reply:
Also, I think we should try to somehow visually highlight our recommended installation method.

Reply:
> A direct "bare metal" installation into should be either the second or potentially even the last option IMO.

The way I laid it out was for "increasing virtualization" as opposed to "most recommended", IMO virtual environment doesn't really make sense first, since it still involves an element of bare metal installation. 

> I think we should try to somehow visually highlight our recommended installation method.

Is that not highlighted by the subsequent Quick Start section?

Reply:
> > A direct "bare metal" installation into should be either the second or potentially even the last option IMO.
> 
> The way I laid it out was for "increasing virtualization" as opposed to "most recommended", IMO virtual environment doesn't really make sense first, since it still involves an element of bare metal installation.

Fair enough.

> > I think we should try to somehow visually highlight our recommended installation method.
> 
> Is that not highlighted by the subsequent Quick Start section?

I think it's important to have a *recommended* way that is clearly marked. But if we can't agree on that, I'm also ok to just leave it *as-is* for now.

Reply:
Yeh I'm going to leave as-is for now. Plus I would note that the line above the panels says: These are shown below, followed by a recommended “quick-install” route on your local computer.